### DISCUSSION TITLE
Backquote (backtick) breaks the edit page.

### DISCUSSION THREAD

**Discussion by junaruga** [community/community#41202] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I searched the past discussions, and as I didn't see this issue, I would post.

I am using the browser Firefox version 106.0.4 on Fedora Linux 37. I see the issue that the backquote (backtick) is not properly displayed on the issue edit page. But it correctly print the code block on the preview page. It was displayed properly before. I am not sure what caused this issue. Is it a known issue? Do you know how to fix it? Thanks.

```
$ rpm -q firefox
firefox-106.0.4-1.fc37.x86_64

$ firefox --version
Mozilla Firefox 106.0.4
```

[1] Edit page - backquote x 3 times is displayed as 1 backquote
![github_firefox_back_quote_edit](https://user-images.githubusercontent.com/121989/206731833-676a73c2-b219-41c1-ba1a-7bd4f9162847.png)

[2] Preview page - the backquote x 3 times works as a code block.
![github_firefox_back_quote_preview](https://user-images.githubusercontent.com/121989/206731855-e7a3b130-5a93-434d-904a-89b09e0e6fe0.png)

**Comment by s-bernard** [comment]
Same here. Backticks have a strange behavior on github issues, using firefox 108.0.

`<>`<>
``
`<>`<>
``
![image](https://user-images.githubusercontent.com/1468889/207958682-c5ddc899-f437-43e3-8f0f-916be85fa14d.png)

It seems it comes from the Noto Sans font. If I force another one, the bug disappear. Here a discussion about it: https://github.com/community/community/discussions/39226
An update seems to exist: https://github.com/notofonts/noto-fonts/issues/736

**Comment by junaruga** [comment, marked as answer]
Thanks for sharing it! This workaround: https://github.com/orgs/community/discussions/39226#discussioncomment-4156008 works for me!

**Comment by dlight** [comment]
This issue is similar to [this one](https://github.com/NixOS/nixpkgs/issues/45621) which wasn't due to Noto Sans but due to `fonts.fontconfig.ultimate.enable` (which I don't have set)

[This comment](https://github.com/NixOS/nixpkgs/issues/45621#issuecomment-415984535) gives a diagnosis which describes perfectly what is happening here: a backtick is becoming a combining grave accent

> A weird way to check if the issue is with the inputted character, or with the rendering of the character could be to copy this sample backtick: `` ` `` and paste it in the same situations, where it would combine with a letter.
> 
> This strangely looks like a [combining grave accent](https://www.fileformat.info/info/unicode/char/0300/index.htm) `̀` which combines with the character to its left `è` instead of `` e` ``.
> 
> > **NOTE** when saving the post, github _could_ realistically replace the combining grave accent with a "real" backtick through unicode normalization. **Only** the first backtick is expected to stay valid after I press save.
> 
> EDIT: neat, the semantics of the characters were kept, so the first one is a backtick, the second one is the combining one (which is hard to select).

**Reply by dlight** [reply]
The monospace workaround works for me too, anyway (maybe it's just because the monospace font isn't Noto Sans?)

**Comment by einenlum** [comment]
Same issue here. It makes it impossible to use Firefox on Github.

Here is a Github element on Chromium:

![cc](https://user-images.githubusercontent.com/5675200/224265480-975bbe31-8f08-4965-96b3-c6adf35f71f1.png)

Here is the same one on Firefox:

![cc2](https://user-images.githubusercontent.com/5675200/224265530-6abedcb9-2ab2-46bf-a001-c06c82bd6817.png)

Using 5.15.94-1-MANJARO and Firefox 110.0.

**Comment by JubilantJerry** [comment]
I encountered this error even though the official Noto Sans file was fixed like 8 years ago. I almost went crazy trying to figure out how to update my system font, which turned out to already be up-to-date.

It turned out that Firefox was using a font file stored in an add-on / extension, rather than my system font. You can see this is happening by using Inspect on the GitHub text box, and then checking the Font section to see what font is being used. If you check All Fonts on Page, you will find the Noto Sans file is located at a weird moz-extension:// url. You can also see if the font issue gets fixed in Private Mode. So disabling the offending add-on fixes the issue, I don't know a good way to identify which add-on can be the cause though so I just tried disabling them one at a time.
